web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/TomeEventSubscriber.php

```diff
@@ -132,17 +133,22 @@ public static function getExcludedDirectories() {
    */
   public function modifyHtml(ModifyHtmlEvent $event) {
     $html = $event->getHtml();
+    $html5 = new HTML5();
 
     // LIBXML_SCHEMA_CREATE fixes a problem wherein DOMDocument would remove closing HTML
     // tags within quoted text in a script element. See https://bugs.php.net/bug.php?id=74628
     $document = new \DOMDocument();
     @$document->loadHTML($html, LIBXML_SCHEMA_CREATE);
+
     $xpath = new \DOMXPath($document);
     $changes = FALSE;
+    $nodes = $xpath->query('//a[starts-with(@href,"/es")]');
+
     /** @var \DOMElement $node */
-    foreach ($xpath->query('//a[(starts-with(@href,"/es"))]') as $node) {
+    foreach ($nodes as $node) {
       $original_href = $node->getAttribute('href');
-      $new_href = NULL;
+      $new_href = '/es/';
```

Comment:
This is a mistake -- and maybe an indication that I should have put a comment here! By setting $new_href to '/es/' here, you're changing *all* of the hrefs that start with "/es" to "/es/". For example, "/es/beneficios-gobierno" changes to "/es/". 

It would be reasonable to set $new_href to $original_href here. The reason I used NULL (and should have added a comment, probably) is that I wanted to avoid re-writing the HTML if none of the hrefs needed to be updated. Doing it this way let me check for that. (Why avoid re-writing the HTML? Mainly just being cautious in case this process might introduce unintended changes. It should help a little with performance too.)

Reply:
Oh no, I'm sorry. I was testing something and I totally forgot to fix this.